doc/index.md

```diff
@@ -10,7 +10,7 @@ width: 125
 
 # Overview
 
-<h2 style="margin-top: 0.3em;">The powerful data exploration and application framework for Python</h2>
+<h1 style="margin-top: 0.3em;">The powerful data exploration and application framework for Python</h1>
```

Comment:
I think the `h2` makes the header small which to me is strange that the first header is not the largest in the page.

But maybe its because you want it to fill up just one line?

Reply:
I reverted it.